### PR TITLE
[android] added styles and custom layouts to the new material settings

### PR DESCRIPTION
**Pull request by marcelinkaaa**
@aldenml styling and adjustments to layouts are complete. I added proper colors to material theme (notice that there is no separate style for preference activity like before - less to maintain)

new layouts such as view_preference_button_action_2.xml - are added for later and will replace view_preference_button_action.xml we have currently - as this layout is active now with original settings activity. 

Next steps:
1. work on making preferences functional ( with @grzesiekrzaca ):
    a) BitTorrent connection, Wifi networks only, Select Storage
        @gubatron - remember talking over the new layouts in Denver? We wanted to replace 'Use 3G/4G for BitTorrent' for 'Wi-Fi Networks Only' where cloud downloads are included in the setting. 
Do you have any comments/pointers for me and @grzesiekrzaca ? 
    b) Toggle state and search settings
    c) Torrent preferences
    d) Other
2. work on the single and double pane view of settings based on screen size

Currently:
screenshot_20161214-185431
screenshot_20161214-185435

### CODE REVIEW

File: android/src/com/frostwire/android/gui/views/preference/ButtonActionPreference.java [hunk 1]
```diff
@@ -80,6 +80,9 @@ public void setOnActionListener(OnClickListener listener) {
 
     @Override
     protected View onCreateView(ViewGroup parent) {
+
+        //for the SettingsActivity2 we will use current
+        // View v = View.inflate(getContext(), R.layout.view_preference_button_action_2, null);
```
Comment (aldenml) [inline android/src/com/frostwire/android/gui/views/preference/ButtonActionPreference.java]: maybe there is a new "ButtonActionPreference" equivalent in preferences support

File: android/res/layout/view_preference_buy_ads_2.xml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+    <TextView
+        android:id="@android:id/summary"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:maxLines="3"
+        android:textSize="@dimen/text_small" />
+
+</LinearLayout>
+
```
Comment (aldenml) [inline android/res/layout/view_preference_buy_ads_2.xml]: space :) (don't bother)

File: android/res/drawable/ad_menu_item_background_filled.xml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+/*
+ * Created by Angel Leon (@gubatron), Alden Torres (aldenml),
+ * Marcelina Knitter (@marcelinkaaa)
+ * Copyright (c) 2011-2016, FrostWire(R). All rights reserved.
```
Comment (aldenml) [inline android/res/drawable/ad_menu_item_background_filled.xml]: new files should have the apache license (no need to change it now)

File: android/src/com/frostwire/android/gui/views/preference/ButtonActionPreference.java (hunk 1 above)
Comment (marcelinkaaa) [inline android/src/com/frostwire/android/gui/views/preference/ButtonActionPreference.java]: Yes. I will look into it tomorrow. If possible would be a lot better.

**Pull request merged**